### DISCUSSION TITLE
Only respect "focus-follows-mouse" on mouse move

### DISCUSSION THREAD

**Discussion by fabolous005** [ghostty-org/ghostty#3595] [Feature Requests, Ideas] [answered]
When enabling `focus-follows-mouse` the `goto_split` property doesn't really work, because the focus immediately jumps back to split where the mouse is.

**Expected behavior**: Ignore `focus-follows-mouse` when switching splits via keyboard and only on mouse move event switch the focus to the respective spltit.

**Comment by mitchellh** [comment, marked as answer]
#3229 

2nd result searching `focus-follows-mouse`. A miracle!

**Reply by fabolous005** [reply]
I am truly sorry for that, i only searched for goto_split.
You guys probably have enough silly questions/tickets to handle...
